src/dmd/errors.d

```diff
@@ -72,6 +95,12 @@ extern (C++) void errorSupplemental(const ref Loc loc, const(char)* format, ...)
     va_end(ap);
 }
 
+/**
+ * Print a warning message, increasing the global warning count.
+ * Params:
+ *      loc = location of error
+ *      format = printf-style format specification
+ */
```

Comment:
Not that I would care much about (and it makes sense in DMD's ugly codebase), but as an FYI we recently defined documentation strings in the [DStyle](https://dlang.org/dstyle.html#phobos_documentation).